pkgs/top-level/aliases.nix

```diff
@@ -1144,6 +1144,7 @@ mapAliases ({
   teleprompter = throw "teleprompter has been removed. reason: upstream dead and does not work with recent electron versions"; # Adedd 2024-03-14
   tensile = throw "'tensile' has been replaced with 'rocmPackages.tensile'"; # Added 2023-10-08
   testVersion = testers.testVersion; # Added 2022-04-20
+  tetrio-desktop = TETR.IO; # Added 2024-03-25
```

Comment:
This won't work because there is no "TETR.IO" package, it is still called `tetrio-desktop`.
Also I think the package naming convention is to have it all lowercase, so `tetr.io`. 

Presonally I don't really see a reason why we should rename the package though :thinking:

Reply:
I don't see why the package should be renamed either. If it is renamed, it seems odd to me for the stylization of the name to be preserved in the package name. `tetrio` would be preferable to me.

Reply:
My concern for backwards compatibility was about the name of the binary (rather than that of the package) changing from `tetrio-desktop` to `TETR.IO`.

Reply:
Right, I undo my commit.